[2025-02-26T13:07:09.343+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-26T13:07:09.405+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: json_ingestion_s3_partitioned.upload_events manual__2025-02-26T13:07:05.320042+00:00 [queued]>
[2025-02-26T13:07:09.418+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: json_ingestion_s3_partitioned.upload_events manual__2025-02-26T13:07:05.320042+00:00 [queued]>
[2025-02-26T13:07:09.419+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-26T13:07:09.441+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): upload_events> on 2025-02-26 13:07:05.320042+00:00
[2025-02-26T13:07:09.455+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'json_ingestion_s3_partitioned', 'upload_events', 'manual__2025-02-26T13:07:05.320042+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmpzi_vviax']
[2025-02-26T13:07:09.458+0000] {standard_task_runner.py:105} INFO - Job 35: Subtask upload_events
[2025-02-26T13:07:09.461+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=491) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-26T13:07:09.463+0000] {standard_task_runner.py:72} INFO - Started process 494 to run task
[2025-02-26T13:07:09.486+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-26T13:07:09.544+0000] {task_command.py:467} INFO - Running <TaskInstance: json_ingestion_s3_partitioned.upload_events manual__2025-02-26T13:07:05.320042+00:00 [running]> on host 5ae8cf5e342a
[2025-02-26T13:07:09.661+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='json_ingestion_s3_partitioned' AIRFLOW_CTX_TASK_ID='upload_events' AIRFLOW_CTX_EXECUTION_DATE='2025-02-26T13:07:05.320042+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-26T13:07:05.320042+00:00'
[2025-02-26T13:07:09.662+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-26T13:07:09.682+0000] {logging_mixin.py:190} INFO - Partition Path: 2025-02-26/
[2025-02-26T13:07:09.683+0000] {logging_mixin.py:190} INFO - Uploading events.json to users_events/2025-02-26/events.json
[2025-02-26T13:07:09.696+0000] {logging_mixin.py:190} INFO - Error uploading events.json: Unable to locate credentials
[2025-02-26T13:07:09.697+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-26T13:07:09.712+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-26T13:07:09.713+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=json_ingestion_s3_partitioned, task_id=upload_events, run_id=manual__2025-02-26T13:07:05.320042+00:00, execution_date=20250226T130705, start_date=20250226T130709, end_date=20250226T130709
[2025-02-26T13:07:09.783+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-26T13:07:09.823+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-26T13:07:09.826+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
